CONTRIBUTING.md

```diff
@@ -0,0 +1,529 @@
+directory.  All interactive tutorials are contained in the top-level directory
+[Tutorials](https://github.com/psi4/psi4numpy/tree/master/Tutorials).  These
+tutorials are organized in logical order of progression, which is enumerated in
+detail
+[here](https://github.com/psi4/psi4numpy/tree/master/Tutorials#interactive-tutorials).
+
+# How can I contribute to Psi4NumPy?
+
+## Reporting Issues
+
+Issues with the Psi4NumPy codebase should be reported through the "New Issue"
+button on the [Issues](https://github.com/psi4/psi4numpy/issues) tab of the
+Psi4NumPy GitHub.  Please be as specific as possible about the operating system
+and Python, Psi4, NumPy, and/or SciPy versions.  Also, a short description of
+the issue along with a minimum working example (MWE) of code which demonstrates
+the issue should be included whenever possible.  A guide to writing good issues
+can be found [here](https://guides.github.com/features/issues/).
+
+## Feature Requests
+
+New features for Psi4NumPy can be requested by raising a new issue (see
+[Reporting Issues](#reporting-issues) above) and changing the label on the
+issue (right hand side of page) to "enhancement".  Then, submit the issue as
+normal.
+
+## Contributing Content
+
+Contributing content to the Psi4NumPy repository assumes the model of many other
+open-source projects, consisting of the following stages:
+
+1. [Creating your own fork of the Psi4NumPy repository](#creating-your-own-fork-of-the-psi4numpy-repository),
+2. [Creating a new feature branch](#creating a new feature branch) for your content,
+3. [Creating and committing your content](#creating-and-committing-your-content),
+4. [Opening a pull request](#opening-a-pull-request) to merge your content into the parent repository,
+5. [Testing and continuing integration](#testing-and-continuous-integration) of your content,
+6. [Code review](#code-review) with the Psi4NumPy developer community, and
+7. [Merging your content](#merging-your-content) into the main Psi4NumPy repository;
+
+each of these stages will be discussed in detail in the [Suggested
+Workflow](#suggested-workflow) section below. WHile this process can seem
+daunting, this section is meant to aid you to more easily make effective,
+meaningful contributions to this repository.  After all, we want your input!  
+
+### Suggested Workflow
+
+This section outlines the recommended workflow for contributing to Psi4NumPy.
+For more information, the suggested workflow for contributing to the Psi4
+repository
+[here](http://psicode.org/psi4manual/1.1/build_obtaining.html#what-is-the-suggested-github-workflow)
+and the GitHub guide [here](https://guides.github.com/introduction/flow/)
+provide graphical representation  which visualizes this process.
+
+Note: For clarity when demonstrating Git commands, we will indicate the
+(current-branch) and relative filepath to the `top-level-psi4numpy-dir` within
+the shell prompt.
+
```

Comment:
Completely agree, we should not write out own Git basics guide here. There are plenty of extant tutorials that we can link to. Unfortunately, we may want to replace everything below with a link. @loriab Any thoughts here?